client/src/main/java/io/pravega/client/control/impl/ModelHelper.java

```diff
@@ -146,8 +146,8 @@ public static final KeyValueTableConfiguration encode(final KeyValueTableConfig
         Preconditions.checkNotNull(config.getKvtName(), "kvtName");
```

Comment:
You may consider adding a Preconditions check for PrimaryKey>0 & SecondaryKey>=0

Reply:
Added checks